scripts/postbuild.ts

```diff
@@ -0,0 +1,61 @@
+/*
+  This script prepares the central `build` directory for NPM package creation.
+  It first copies all non-code files into the `build` directory, including `package.json`, which
+  is edited to adjust entry point paths. These corrections are performed so that the paths align with
+  the directory structure inside `build`.
+*/
+
+import * as fs from 'fs';
+
+import * as path from 'path';
+
+const BUILD_DIR = 'build';
+const ASSETS = ['README.md', 'LICENSE', 'package.json', '.npmignore'];
+const ENTRY_POINTS = ['main', 'module', 'types'];
+
+// check if build dir exists
+try {
+  if (!fs.existsSync(path.resolve(BUILD_DIR))) {
+    console.error(`Directory ${BUILD_DIR} DOES NOT exist`);
+    console.error("This script should only be executed after you've run `yarn build`.");
+    process.exit(1);
+  }
+} catch (error) {
+  console.error(`Error while looking up directory ${BUILD_DIR}`);
+}
+
+// copy non-code assets to build dir
+ASSETS.forEach(asset => {
+  const assetPath = path.resolve(asset);
+  try {
+    if (!fs.existsSync(assetPath)) {
+      console.error(`Asset ${asset} does not exist.`);
+      process.exit(1);
+    }
+    fs.copyFileSync(assetPath, path.resolve(BUILD_DIR, asset));
+  } catch (error) {
+    console.error(`Error while copying ${asset} to ${BUILD_DIR}`);
+  }
+});
+
+// package.json modifications
+const packageJsonPath = path.resolve(BUILD_DIR, 'package.json');
+const pkgJson: { [key: string]: string } = require(packageJsonPath);
```

Comment:
This is not only string -> string, the values can be anything. I would define the value as `unknown` and then explicitly type out the entry points.

Reply:
Also good catch - changed it.